tests/python/relay/test_executor.py

```diff
@@ -44,12 +44,15 @@ def test_attr_check():
 
 
 def test_create_executor_not_found():
-    with pytest.raises(TVMError, match='Executor "woof" is not defined'):
+    with pytest.raises(TVMError, match='Executor "woof" is not defined.'):
         Executor("woof", {})
 
 
 def test_create_executor_attr_not_found():
-    with pytest.raises(TVMError, match='Attribute "woof" is not available on this Executor'):
+    with pytest.raises(
+        TVMError,
+        match='TVMError: Executor "aot": Attribute "woof" not in.*workspace-byte-alignment, unpacked-api, interface-api, link-params',
```

Comment:
i think you may just need an extra `.*` at the end of the match string here

Reply:
Hi @areusch thanks for the hint! I have tried it with a .* at the end now. It is a bit hard for me to test this, because locally the test passes  in all the different versions I have tried so far.

Reply:
hmmm. have you tried running locally via e.g.
```
docker/bash.sh ci_gpu python3 -mpylint tests/python/relay/test_executor.py
```

i think maybe now what's happening is that you've got `-` in your regex `match=` and you need to escape those with https://docs.python.org/3/library/re.html#re.escape

i'm not sure why this wouldn't happen on your version of pytest tho.

Reply:
Hi, thanks again. I currently try to build the docker behind our proxy...

Reply:
is the image too large to download? you should be able to just pull the container we use in CI. docker/bash.sh should do this.

Reply:
Hi, when I do this directly, it results in 
docker/bash.sh ci_cpu python3 -mpylint tests/python/relay/test_executor.py
REPO_DIR: home/xxx/MyGithub/tvm
DOCKER CONTAINER NAME: tlcpack/ci-cpu:v0.83

Running 'python3 -mpylint tests/python/relay/test_executor.py' inside tlcpack/ci-cpu:v0.83...
mesg: ttyname failed: Inappropriate ioctl for device
Adding group `xxx' (GID 1683538626) ...
Done.
/usr/bin/python3: No module named pylint

I understand that this should work for python tests without rebuilding it. I just tried to rebuild the docker container after this failed. But out of curiosity: For C++ tests it must be rebuilt, right?

Reply:
whooops, i'm really sorry, i sent you the wrong command. it should be 
```
docker/bash.sh ci_cpu python3 -mpytest tests/python/relay/test_executor.py
```

Reply:
Hi, 
this also fails with
OSError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /HOME/MyGithub/tvm/build/libtvm.so)
Sorry I am not experienced with docker. Is it looking for glibc on my machine or in the docker agent?

Reply:
@SebastianBoblestETAS that seems odd....i don't think you should see GLIBC errors from inside the docker container unless we've misconfigured the container. could you send the full log and maybe i can see if something else is happening in trying to launch the container? though it seems like that part worked previously for you